brand/README.md

```diff
@@ -0,0 +1,94 @@
+<p align="center"><img src="primary-logo.png" width="360"></p>
+<p align="center"><b>Cloud Native Runtime Security.</b></p>
+
+# Falco Branding Guidelines
+
+This document describes The Falco Project's branding guidelines, language, and message.
+
+Content in this document can be used to publically share about Falco.
+
+
+
+### Logo
+
+There are 3 logos available for use in this directory. Use the primary logo unless required otherwise due to background issues, or printing.
+
+The Falco logo is Apache 2 licensed and free to use in media and publication for the CNCF Falco project.
+
+### Slogan
+
+> Cloud Native Runtime Security
+
+### What is Falco?
+
+Falco is a Runtime Security project originally built by Sysdig, Inc.
+Falco was donated to the CNCF in October 2018.
```

Comment:
I thought the correct word is 'contributed' vs. 'donated'

Reply:
I checked the CNCF website guidelines and it looks like `created by` is what we were looking for.

Reference: https://github.com/cncf/foundation/blob/master/website-guidelines.md